lib/gitalytics.rb

```diff
@@ -33,7 +33,9 @@ def output_html_report(data, open_in_browser)
     erb = ERB.new(template_file)
     output_file = "gitalytics_result.html"
     File.open(output_file, 'w+') do |file|
-      @users = data[:users]
+      @users = data[:users].sort do |x, y|
+				y.commits.length <=> x.commits.length
+			end
```

Comment:
`end` at 38, 3 is not aligned with `@users = data[:users].sort do |x, y|` at 36, 6.<br>Tab detected.